pom.xml

```diff
@@ -27,16 +27,13 @@
     <openapi-generator.version>5.3.1</openapi-generator.version>
     <mapstruct.version>1.4.1.Final</mapstruct.version>
     <json.version>20200518</json.version>
-    <hazelcast.version>4.0.2</hazelcast.version>
     <spring-kafka.version>2.8.2</spring-kafka.version>
     <wiremock.version>2.27.2</wiremock.version>
     <spring-kafka-test.version>2.8.2</spring-kafka-test.version>
     <embedded.db.version>2.1.1</embedded.db.version>
     <pubsub.client.version>2.0.0</pubsub.client.version>
     <jackson.version>2.11.1</jackson.version>
     <json.path.version>2.5.0</json.path.version>
```

Comment:
This line downgrades json-path from 2.7.0 to 2.5.0.

Try `mvn dependency:tree -Dincludes=com.jayway.jsonpath:json-path`:

```
[INFO] org.folio:mod-remote-storage:jar:1.6.0-SNAPSHOT
[INFO] \- com.jayway.jsonpath:json-path:jar:2.5.0:compile
```

After removing

```
<json.path.version>2.5.0</json.path.version>
```
and
```
<version>${json.path.version}</version>
```

running `mvn dependency:tree -Dincludes=com.jayway.jsonpath:json-path` yields

```
[INFO] org.folio:mod-remote-storage:jar:1.6.0-SNAPSHOT
[INFO] \- com.jayway.jsonpath:json-path:jar:2.7.0:compile
```